include/freerdp/settings.h

```diff
@@ -417,6 +418,12 @@ typedef struct rdp_monitor rdpMonitor;
 #define RDPDR_DTYP_FILESYSTEM		0x00000008
 #define RDPDR_DTYP_SMARTCARD		0x00000020
 
+/* TODO: OUTCH! cf. https://github.com/FreeRDP/FreeRDP/issues/4833
```

Comment:
Does not belong to this pull, remove.
Pull requests for `2.0.0 + 1` are of course welcome.